python_backend/api/v1/endpoints/quality.py

```
@@ -0,0 +1,135 @@
+        if sentence_count > 5:
+            suggestions.append("긴 문단을 여러 개의 짧은 문단으로 나누는 것을 고려해보세요")
+            
+        return QualityAnalysisResponse(
+            grammarScore=grammar_score,
+            formalityScore=formality_score,
+            readabilityScore=readability_score,
+            suggestions=suggestions
+        )
+        
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"품질 분석 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)